client/src/analysis/components/AnalysisLayoutThreeCol.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import { Grid, Header, Icon, Segment } from "semantic-ui-react";
+
+interface AnalysisLayoutTwoColProps {
+    title: string,
+    subtitle: React.ReactNode,
+    left: React.ReactNode,
+    mid: React.ReactNode,
+    right: React.ReactNode,
+    toolbar?: React.ReactNode,
+}
+
+type MergedProps = AnalysisLayoutTwoColProps;
+const title2 = "Masking out of zero-order diffraction peak in real space";
```

Comment:
Why not have these three titles as props, so this three-column layout can be re-used by other analyses?